jenkins-pipelines/perf-regression-alternator-throughput-30min-ttl.jenkinsfile

```diff
@@ -0,0 +1,12 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
```

Comment:
@yarongilor was this run ? was the results shared ? is it planned to be run on a weekly basis ?